modules/oxide-components/src/test/ts/browser/components/Textarea.spec.tsx

```diff
@@ -0,0 +1,203 @@
+import { userEvent } from '@vitest/browser/context';
+import { Textarea } from 'oxide-components/components/textarea/Textarea.component';
+import type { Height } from 'oxide-components/components/textarea/Textarea.utils';
+import { classes } from 'oxide-components/utils/Styles';
+import { useState } from 'react';
+import { describe, expect, it } from 'vitest';
+import { render } from 'vitest-browser-react';
+
+describe('Textarea', () => {
```

Comment:
Could we name this `browser.TextAreaTest` to follow the conventions we have elsewhere or is that some how not ideal if so why?